src/OutboundServer/handlers.js

```diff
@@ -47,6 +48,12 @@ const handleAccountsError = (method, err, ctx) =>
  * Handler for outbound transfer request initiation
  */
 const postTransfers = async (ctx) => {
+    const histTimerEnd = Metrics.getHistogram(
+        'outbound_post_transfers',
```

Comment:
Same comment here about the Route Maps having a duplicate metric name, and the rest of the entries below?

Reply:
decided not to go with this option as name in route map requires the path to access